plugin/teksi_wastewater/interlis/interlis_importer_exporter.py

```diff
@@ -182,6 +184,8 @@ def interlis_export(
             labels_file_path = os.path.join(tempdir.name, "labels.geojson")
             self._export_labels_file(
                 limit_to_selection=limit_to_selection,
+                # neu export orientation
+                export_orientation=export_orientation,
```

Comment:
Around line 195 `self._export_to_intermediate_schema(` also needs this parameter to pass it to `InterlisExporterToIntermediateSchema`

Reply:
@domi4484 Is this better now?